docs/content/latest/yugabyte-platform/install-yugabyte-platform/install-software/kubernetes.md

```diff
@@ -108,34 +108,33 @@ You install Yugabyte Platform on a Kubernetes cluster as follows:
     yw-test-yugaware-ui   LoadBalancer   10.111.241.9   34.93.169.64   80:32006/TCP,9090:30691/TCP   2m12s
     ```
 
-## Customization
+## Customize Yugabyte Platform
 
-1. To change CPU & memory resources:
+You can customize Yugabyte Platform on a Kubernetes cluster in a number of ways, such as the following:
 
-    ```sh
-    helm install yw-test yugabytedb/yugaware -n yb-platform \
-      --set yugaware.resources.requests.cpu=2 \
-      --set yugaware.resources.requests.memory=4Gi \
-      --set yugaware.resources.limits.cpu=2 \
-      --set yugaware.resources.limits.memory=4Gi
-    ```
+- You can change CPU and memory resources by executing a command similar to the following:
 
-1. To disable the internet/public facing LB, provide the annotations to YW service for disabling the public-facing LB.
+  ```sh
+  helm install yw-test yugabytedb/yugaware -n yb-platform \
+    --set yugaware.resources.requests.cpu=2 \
+    --set yugaware.resources.requests.memory=4Gi \
+    --set yugaware.resources.limits.cpu=2 \
```

Comment:
Let's include an example of customizing prometheus memory by adding something like
`prometheus.resources.requests.mem=6Gi`

Reply:
@iSignal I added a second example, but the command syntax is slightly different, so I'm not sure it's correct. Could you, please, check and let me know:

_You can change CPU and memory resources by executing a command similar to one of the following:_

```
helm install yw-test yugabytedb/yugaware -n yb-platform \
  --set yugaware.resources.requests.cpu=2 \
  --set yugaware.resources.requests.memory=4Gi \
  --set yugaware.resources.limits.cpu=2 \
  --set yugaware.resources.limits.memory=4Gi
```

```
**helm install --namespace yw-test yw-test yugabytedb/yugaware 
  --set yugaware.resources.requests.memory=3Gi, prometheus.resources.requests.cpu=3**
```

Reply:
You can collapse them into one example like below. 

``` 
helm install yw-test yugabytedb/yugaware -n yb-platform \
  --set yugaware.resources.requests.cpu=2 \
  --set yugaware.resources.requests.memory=4Gi \
  --set yugaware.resources.limits.cpu=2 \
  --set yugaware.resources.limits.memory=4G \
  --set prometheus.resources.requests.mem=6Gi
```